src/Sulu/Bundle/AdminBundle/Tests/Functional/Metadata/FormMetadata/FormMetadataProviderTest.php

```diff
@@ -50,4 +50,59 @@ public function testGetMetadataFromStructureLoader()
         $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
         $this->assertCount(2, $typedForm->getForms());
     }
+
+    public function testGetMetadataTagFiltered()
+    {
+        $typedForm = $this->formMetadataProvider->getMetadata(
+            'page',
+            'de',
+            ['tags' => ['test' => ['value' => 'test-value']]]
+        );
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(1, $typedForm->getForms());
+        $this->assertEquals(['default'], array_keys($typedForm->getForms()));
+
+        $typedForm = $this->formMetadataProvider->getMetadata(
+            'page',
+            'de',
+            ['tags' => ['test' => ['value' => 'test-value2']]]
+        );
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(0, $typedForm->getForms());
+
+        $typedForm = $this->formMetadataProvider->getMetadata(
+            'page',
+            'de',
+            ['tags' => ['test2' => ['value' => 'test-value2']]]
+        );
+        $this->assertInstanceOf(TypedFormMetadata::class, $typedForm);
+        $this->assertCount(0, $typedForm->getForms());
+
+        $typedForm = $this->formMetadataProvider->getMetadata(
+            'page',
+            'de',
+            ['tags' => ['test2' => ['exists' => true, 'test' => 'test-value2']]]
```

Comment:
Did that `exists` slip? Or does it have a purpose?

Reply:
no it slipped :)